sdk/proto/phpfpm.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+package f5.nginx.agent.sdk;
+
+import "gogo.proto";
+
+option go_package = "github.com/nginx/agent/sdk/v2/proto;proto";
+
+// PhpFpmDetails registers a phpfpm master process + child pools
+message PhpFpmDetails {
+  string type = 1 [(gogoproto.jsontag) = "type"];
```

Comment:
Can you add comments to describe what each of these fields are?

Reply:
Added.